website/docs/language/modules/develop/providers.html.md

```diff
@@ -201,38 +215,25 @@ module "tunnel" {
 }
 ```
 
-The subdirectory `./tunnel` must then contain _proxy configuration blocks_ like
-the following, to declare that it requires its calling module to pass
-configurations with these names in its `providers` argument:
+The subdirectory `./tunnel` must then declare the configuration aliases for the
+provider to pass configurations with these names in its `providers` argument:
```

Comment:
This is a somewhat awkward sentence (because it's a confusing concept), it looks a bit like the provider itself has to pass in configurations, 

Maybe something along the lines of:
```suggestion
provider so the calling module can pass configurations with these names in its `providers` argument:
```

Or from the old example "to declare that it requires its calling module" (I'm not even sure if it's required or nah!)